### ISSUE TITLE
Wifi networking NetVM crashes with poor UX

### ISSUE THREAD

**Issue by bnvk** [QubesOS/qubes-issues#1595]
Recently my wifi NetVM crashed and was failed to show any networks where it was previously showing things. I had to shutdown all my VMs and restart the NetVM to get it to discover and connect. My first attempt to shutdown the `sys-net` VM the following notification showed up: 

```
Error starting VM 'sys-net': Failed to kill old QubesDB instance (PID 1958) Terminate it manually and retry. If that isn't QubesDB process remove the pidfile: /var/run/qubes/qubesdb.sys-net.pid
```

After a few attempts to "Kill" the VM it finally shutdown. That error message is quite technical and probably unhelpful to less technical users. However, if I understand the message correctly, it's offering instructions for CLI skilled users to further research the issue, correct?

- Hide that sort of message and tell user to `Please attempt to kill this VM again` 
- Provide a link to more detailed explanation in local help Docs to the user

**Comment by marmarek** [comment]
Generally this error message is an workaround for another bug, related to #1470 . So the proper solution would be fixing that original bug...

**Comment by marmarek** [comment]
Fixed in #1745
